torch/csrc/jit/passes/peephole.cpp

```diff
@@ -14,359 +16,441 @@ static bool mustBeEqual(const c10::optional<T>& a, const c10::optional<T>& b) {
   return a == b && a.has_value();
 }
 
-// The intent for this optimization pass is to catch all of the small, easy to
```

Comment:
It would be easier to review this if you separated the formatting into a separate PR

Reply:
Unfortunately, it's not just formatting, I had to move code chunks around since I added a new class and some peephole opts need to be done before an alias check and some need to be done afterwards (i.e. another code moves). So separating formatting probably won't help that much

Reply:
I had to do something similar in the past, and I made a ghstack with the first PR moving the function to a class

Reply:
just chiming in to say that I agree that in the future, let's separate the simple "mechanical" change with many lines out, and leave the important changes for a smaller follow on diff that can be reviewed more carefully. It's what ghstack is for!